src/domain/app/i18n/en.json

```
@@ -1887,8 +1887,8 @@
     },
     "personsAddedToWaitingListModal": {
       "buttonClose": "Close",
-      "text": "Please note that not all registrants' places are guaranteed. Entrants in the reserve position will only be confirmed if more places become available in the queue. You will receive a separate email from confirmed registration.",
-      "title": "Entrants have been added to the reservation queue"
+      "text": "You will receive a separate confirmation message about your registration to your e-mail, if a place becomes available for you.",
+      "title": "You are registering for the event to the reservation queue"
```

```suggestion
      "title": "You are registering to the event's reservation queue"
```
Just a suggestion - you can also ignore this

---

Changed as suggested. I did similar change to Linked Registration UI in https://github.com/City-of-Helsinki/linkedregistrations-ui/pull/93/files